tests/test_nurbs.py

```diff
@@ -199,6 +200,36 @@ def test_nurbs_translate_nurbs_surface(
     assert_results_equal(get_corresponding_reference_file_path(), input_file)
 
 
+def test_nurbs_cylindrical_shell_sector(
+    assert_results_equal, get_corresponding_reference_file_path
+):
+    """Test the creation of a 3-dimensional cylindrical shell sector."""
+
+    # Create the surface of a quarter of a hollow cylinder
+    surf_obj = create_nurbs_cylindrical_shell_sector(
+        2.3, np.pi / 3, 1, n_ele_u=3, n_ele_v=5
```

Comment:
```suggestion
        2.3, np.pi / 3, 1.7, n_ele_u=3, n_ele_v=5
```
Having a "not so nice" number here avoids cases where we accidentally produce the correct result.